types/fg-loadcss/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for fg-loadcss 3.1
+// Project: https://github.com/filamentgroup/loadCSS
+// Definitions by: Noah Overcash <https://github.com/ncovercash>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function loadCSS(href: string, before?: HTMLElement, media?: string, attributes?: Record<string, string>): HTMLLinkElement;
+
+export function onloadCSS(stylesheet: HTMLLinkElement, callback: () => void): void;
```

Comment:
this one should be probably defined on window:
```ts
declare namespace global {
   interface Window {
        onloadCSS(....
````
it's optional, not part of exported UMD:
https://github.com/filamentgroup/loadCSS/blob/master/src/onloadCSS.js#L2
https://unpkg.com/fg-loadcss@3.1.0/src/loadCSS.js

Reply:
Gotcha, thanks!